plugin/audit_log/audit_log.c

```diff
@@ -68,6 +68,8 @@ static char *audit_log_include_accounts= NULL;
 static char *audit_log_exclude_commands= NULL;
 static char *audit_log_include_commands= NULL;
 
+volatile int64 audit_log_buffer_size_overflow = 0;
```

Comment:
Atomic vars have little to do with volatile; SHOW_LONG is actually unsigned, which is OK for us:
```suggestion
uint64 audit_log_buffer_size_overflow = 0;
```